openedx/adg/lms/webinar/models.py

```diff
@@ -0,0 +1,53 @@
+"""
+All models for webinar app
+"""
+from django.contrib.auth.models import User
+from django.core.validators import FileExtensionValidator
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .constants import ALLOWED_BANNER_EXTENSIONS
+
+
+class Webinar(models.Model):
+    """
+    Model to create a webinar
+    """
+    start_time = models.DateTimeField(verbose_name=_('Start Time'), )
+    end_time = models.DateTimeField(verbose_name=_('End Time'), )
+
+    subject = models.CharField(verbose_name=_('Subject'), max_length=150, )
+    title = models.CharField(verbose_name=_('Title'), max_length=150, )
+    description = models.TextField(verbose_name=_('Description'), )
+    presenter = models.ForeignKey(User, verbose_name=_('Presenter'),
+                                  on_delete=models.CASCADE, related_name='webinar_presenter')
+    meeting_link = models.URLField(default='', verbose_name=_('Meeting Link'), blank=True, )
+    banner = models.ImageField(
+        upload_to='webinar/banners/', verbose_name=_('Banner'),
+        validators=[FileExtensionValidator(ALLOWED_BANNER_EXTENSIONS)],
+        help_text=_('Accepted extensions: .png, .jpg, .svg'),
+    )
+    language = models.CharField(verbose_name=_('Language'), max_length=150, )
+    co_host = models.ManyToManyField(User, verbose_name=_('Co-Host'), blank=True, related_name='webinar_co_host')
+    panelist = models.ManyToManyField(User, verbose_name=_('Panelist'), blank=True, related_name='webinar_panelist')
+
+    CONFIRM = 'confirm'
+    CANCELLED = 'cancelled'
+    STATUS_CHOICES = (
+        (CONFIRM, _('Confirm')),
+        (CANCELLED, _('Cancelled'))
+    )
+    status = models.CharField(
+        verbose_name=_('Webinar Status'), choices=STATUS_CHOICES, max_length=10, default=CONFIRM,
+    )
+    is_virtual = models.BooleanField(default=True, verbose_name=_('Virtual Event'))
+    created_by = models.ForeignKey(User, verbose_name=_('Created By'),
+                                   on_delete=models.CASCADE, related_name='webinar_created_by')
+    last_modified_by = models.ForeignKey(User, verbose_name=_('Last Modified By'), on_delete=models.CASCADE,
+                                         null=True, blank=True, related_name='webinar_modified_by')
+
```

Comment:
Shouldn't there be a field for rsvp/respondents, as per the requirements. I believe, at minimum, we'd need a single field with many to many relationship with User. We would require list of all users that registered for an event, for example to email them upon cancellation. It also states in the flow: "Cancellation email sent to users who RSVP'd yes". We'd also need to send email upon event updation, or for reminders, to those who responded "yes"/registered for the event.

Please come up with an appropriate field name and add it in the model. Also remake migrations.